src/sbl_filing_api/entities/models/dto.py

```diff
@@ -7,10 +7,10 @@
 
 class UserActionDTO(BaseModel):
     id: int | None = None
-    user_id: str
-    user_name: str
-    user_email: str
-    timestamp: datetime
+    user_id: str = Field(None, max_length=36)
+    user_name: str = Field(None, max_length=255)
+    user_email: str = Field(None, max_length=255)
```

Comment:
`None` shouldn't be in the `Field(...)` either, I was previously saying to do `Field(..., max_length=255)`, but actually we should be able to just omit the default value if I'm not mistaken, so these could just become `Field(max_length=...)`, give that a try to confirm.